ownCloud/Theming/ThemeCollection.swift

```diff
@@ -135,6 +141,9 @@ class ThemeCollection : NSObject {
 	@objc var statusBarStyle : UIStatusBarStyle
 	@objc var barStyle : UIBarStyle
 
+	// MARK: - SearchBar
+	@objc var searchbarColors : ThemeColorCollection
```

Comment:
For consistency with for example `statusBarStyle`, I'd suggest renaming this to `searchBarColors`.

Reply:
Done